src/test/java/api/support/builders/RequestBuilder.java

```diff
@@ -40,6 +36,8 @@
   private final DateTime cancelledDate;
   private final Integer position;
   private final UUID pickupServicePointId;
+  private final Tags tags;
```

Comment:
In reference to my other comment, I think this could just be a list, rather than a new class.

```suggestion
  private final List<String> tags;
```